packages/itwinui-react/src/core/Surface/Surface.tsx

```diff
@@ -29,7 +33,53 @@ const getSurfaceElevationValue = (elevation: SurfaceProps['elevation']) => {
   }
 };
 
+// ----------------------------------------------------------------------------
+// Surface.Header component
+
+export type SurfaceHeaderProps = React.ComponentPropsWithRef<'div'>;
+
+const SurfaceHeader = React.forwardRef<HTMLDivElement, SurfaceHeaderProps>(
```

Comment:
Jsdocs missing for new components

Reply:
and jsdoc should go at the bottom (near line 124)

Reply:
Jsdocs for new subcomponents have been written